api-samples/privacy/service-worker.js

```diff
@@ -0,0 +1,50 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+chrome.runtime.onInstalled.addListener(() => {
+  // Set default value for Autofill Enabled
+  chrome.privacy.services.autofillEnabled.set({ value: true });
+});
+
+chrome.runtime.onStartup.addListener(() => {
+  getAutofillEnabledStatus();
+});
+
+chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
+  if (changeInfo.status === 'complete' && tab.active) {
+    getAutofillEnabledStatus();
+  }
+});
+
+function getAutofillEnabledStatus() {
```

Comment:
nit: the method is called `getAuto...` but it actually updates the action. Would be good to relect this in the method name.